heron/cli/src/python/args.py

```diff
@@ -14,7 +14,7 @@
 ''' args.py '''
 import os
 
-import heron.common.src.python.utils as utils
+import heron.common.src.python.utils.config as utils
```

Comment:
This import is vey unintuitive - as it imports config as utils. Can we make it more readable?